listener/listener.go

```diff
@@ -128,6 +131,20 @@ func WithFilterDescriber(filterName string, filter servicebus.FilterDescriber) M
 	}
 }
 
+//allos listeners to control subscription details for longer lived operations.
+//If you using RetryLater you probably want this
```

Comment:
did so.

Reply:
I saw you had something there, but I was actually meaning adding this info in addition to what you already have. I think it would be good if someone was going to use RetryLater that they were directed to also set WithSubscriptionDetails, especially maxDelivery